config.yml

```diff
@@ -32,3 +34,4 @@ style:
       green: 1.0
       blue: 0.9019
     master_template_id: '1LYS5qE4nV9jzcYw5sXwCza25slDfRA1CIg3cs-hCdpU'
+    strict_validation: true
```

Comment:
Rather then adding it as a config parameter, what are your thoughts on making it an optional CLI argument? This is once again a suggestion, keeping issue #447 in mind. We'll probably be eliminating this whole Googlesheets customization section from the config file eventually, so I'm not quite sure this is the best place to allow this type of configuration.

Reply:
Good question. My use-case is not actually the CLI, but for the app, which is why I added it as a config option. IMO, it makes more sense to modify a config file rather than `metadataModelFuns.py` or `app.R`. But we could change this to config in www/config.json in the shiny app instead, maybe? 

Either way, I agree that this should _also_ be a CLI argument.

Reply:
Ah, I see, that makes sense. We can add the option to schematic's config.yml file (the way you have now) and the `www/config.json` file or the Data Curator App. But it probably makes more sense to add it here because schematic reads all configuration information from this config.yml file, and there are cases when users might not choose to use the Data Curator App to leverage schematic's functionalities, but just rely on the CLI.

Reply:
@sujaypatil96 and @allaway let's keep this as parameter exposed in the config.yml file for now. I agree exposing it as an optional CLI parameter makes a lot of sense as well (and respectively in the corresponding REST API). 

It looks like we may want to introduce a separate manifest config/profile in the future that allows setting style and other manifest parameters. Let's discuss the more general topic of profiles during code review - @sujaypatil96  could you add it to the agenda for next week?

Reply:
ow I agree this looks good to go!

Reply:
😊

Reply:
@milen-sage: yup, let's expose this as an optional parameter in a future CLI update PR.

> It looks like we may want to introduce a separate manifest config/profile in the future that allows setting style and other manifest parameters. Let's discuss the more general topic of profiles during code review - @sujaypatil96 could you add it to the agenda for next week?

Sounds good, I'll add it to the agenda. Looks like we have quite a few items to discuss during our next code review.

> ow I agree this looks good to go! Thanks @allaway! Btw @sujaypatil96 could you add Robert to the contributor's list on schematic docs? Robert has been driving a bunch of features now, in addition to adding code + testing and providing feedback.

Absolutely! I agree @milen-sage. I'll add @allaway to the contributors section in an upcoming documentation update PR?